CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
    * FIXED: Add missing GEOS:GEOS dep to mjolnir target [#2901](https://github.com/valhalla/valhalla/pull/2901)
 
 * **Enhancement**
+   * Updated max_matrix_locations parameter description. [#2983](https://github.com/valhalla/valhalla/pull/2983)
```

Comment:
newest changes are towards the bottom of the list.

Reply:
On second thoughts this change probably does not need a changelog entry since its trivial.

Reply:
Moved new line to the bottom.

A also thought that this is a tiny update so it doesn't need to go to changelog. But without changelog entry this PR doesn't pass Changelog Validation ci test.

Reply:
Don't worry about CI failing for that, it's more meant as a gentle reminder to review whether a changelog entry is needed or not. I'd also say this is one of the cases where that's a 'not'.

Reply:
Thank you for explanation) I removed new line from the Changelog.